### ISSUE TITLE
Incorrect AllPortals value?

### ISSUE THREAD

**Issue by johnd0e** [TheSned/IITCPlugins#17]
https://github.com/TheSned/IITCPlugins/blob/b52007347b1abab2b5be95c19f7f6ac0bb3d8aff/force-zoomlevel.user.js#L103

Shouldn't it be `15`?

**Comment by TheSned** [comment]
Yes, you are correct. However, zoom level 15 functions the same as 17, so in this case it doesn't matter too much.

**Comment by Omen75** [comment]
I just did an uninstall of all plugins and trying to re-install and the force zoom is not working anymore... not sure if it's because of the updates or i've had multiple versions of iitc loaded and now reloaded only the most recent.? anyway i find this plugin useful and hope to seeit working again.
